Snowplow iOSTests/TestGeneratedJsons.m

```diff
@@ -93,6 +94,28 @@ - (void)testGeoContextJson {
     XCTAssertTrue([validator validateJson:json]);
 }
 
+- (void)testGDPRContextJson {
+    SPTracker * tracker = [self getTracker:@"acme.fake.url"];
+    SNOWGlobalContexts * sgc = [tracker globalContexts];
+    SPPayload * payl = [[SPPayload alloc]initWithNSDictionary:@{}];
+    NSArray * contexts = [sgc evaluateWithPayload:payl];
+    // FIXME: The following values are duplicated in the builder in getTracker
```

Comment:
You can likely now use the declared keys in Snowplow.h to remove this FIXME note?

Reply:
Good point!

Reply:
Although this is referring to the values not the keys :-)

Reply:
Ahh understood!